### DISCUSSION TITLE
Welcome to OpenPharmMDFlow Discussions!

### DISCUSSION THREAD

**Discussion by mikemhenry** [omsf/OpenPharmMDFlow#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amburan** [comment]
Awesome! Thanks for creating this @mikemhenry ! I am a computational modeler at Bristol Myers Squibb. I am keen on increasing the use of open-source MD tools at BMS
